src/ingest_validation_tools/table-schemas/level-2/slideseq.yaml

```diff
@@ -0,0 +1,89 @@
+#doc_url: TODO
+fields:
+  # Overrides of Level 1 fields:
+  -
+    name: assay_category
+    constraints:
+      enum:
+        - sequence
+  -
+    name: assay_type
+    constraints:
+      enum: [] # TODO
+  -
+    name: analyte_class
+    constraints:
+      enum:
+        - RNA
+
+  # Level 2 fields:
+  -
+    heading: Level 2
+    name: acquisition_instrument_vendor
+    description: An acquisition_instrument is the device that contains the signal detection hardware and signal processing software. Assays can generate signals such as light of various intensities or color or signals representing molecular mass.
+  -
+    name: acquisition_instrument_model
+    description: Manufacturers of an acquisition instrument may offer various versions (models) of that instrument with different features or sensitivities. Differences in features or sensitivities may be relevant to processing or interpretation of the data.
+  -
+    name: rnaseq_assay_input
+    description: Number of cell/nuclei input to the assay
```

Comment:
```suggestion
 delete
```

Reply:
@cebriggs7135 : For deletions, select the entire range you want deleted, and delete all the lines in the suggestion block, so the closing <code>&#x60;&#x60;&#x60;</code> immediately follows the opening <code>&#x60;&#x60;&#x60;</code>.

Reply:
resolved separately.